tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
     "compilerOptions": {
         "target": "es5",
+        "outDir": "lib/",
         "module": "commonjs",
-        "noEmitHelpers": true,
+        "noEmitHelpers": false,
```

Comment:
That's why I removed the snippet:
You have removed
```js

  // provide scoped __extends for TypeScript's `extend` keyword so it will not provide global one during compilation
  function __extends(d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
  }
```
Setting this to false emits this snippet automatically at the right place.